.github/actions/install-dashboards/action.yml

```diff
@@ -0,0 +1,81 @@
+name: 'Install Dashboards with Plugin'
+description: 'Installs OpenSearch Dashboard with a Plugin from github, then checkouts the correct dashboards version for the plugin, configures npm/yarn, and bootstraps Dashboards'
+
+inputs:
+  plugin_name:
+    description: 'The the name of the plugin to use, such as security-dashboards-plugin'
+    required: true
+
+outputs:
+  dashboards-directory:
+    description: "The directory where the dashboards has been configured"
+    value: ${{ steps.determine-dashboards-directory.outputs.dashboards-directory }}
+
+  plugin-directory:
+    description: "The directory where the plugin has been configured"
+    value: ${{ steps.determine-plugin-directory.outputs.plugin-directory }}
+
+
+runs:
+  using: "composite"
+  steps:
+    - id: determine-dashboards-directory
+      run: echo "dashboards-directory=OpenSearch-Dashboards"  >> $GITHUB_OUTPUT
+      shell: bash
```

Comment:
Looks like `bash` is available on all platforms according to this: https://dev.to/pwd9000/github-actions-all-the-shells-581h

Nice use of a composite action! 🙌